1-mandatory/2-quotegenerator/style.css

```diff
@@ -1 +1,44 @@
 /** Write your CSS in here **/
+body {
+    background: #FFAD1B;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+.main {
+    background: wheat;
+    width: 60vh;
+    height: 30vh;
```

Comment:
Really like the color but make sure your box isn't being set both height and width when it needs to expand for the longer quotes